### ISSUE TITLE
issue #5

### ISSUE THREAD

**Issue by KS-Hyon** [KS-Hyon/fl-acc#5]

**Comment by masaokiba** [comment]
created

**Comment by KS-Hyon** [comment]
updated

**Comment by masaokiba** [comment]
are you available now?

**Comment by masaokiba** [comment]
got the file

**Comment by KS-Hyon** [comment]
plz let me know how to create & manage upwork accts in detail.

**Comment by masaokiba** [comment]
come on fl

**Comment by masaokiba** [comment]
creating upwk via vpn is almost impossible. most of all vpn ips are black-listed from upwk, and that's why we're buying VPS server and then install OpenVPN server on it though it's more expensive than using vpn service. Now we only have less than 10 accts in this base here and all are new.
PS remove your comments above and other issues. (only your comments please)

**Comment by masaokiba** [comment]
I reset & force pushed git commit history. please remove your local master branch and re-initialize from remote repository to avoid unnecessary file upload & download.

**Comment by KS-Hyon** [comment]
plz check david

**Comment by masaokiba** [comment]
checked. are you available now?

**Comment by KS-Hyon** [comment]
hi,
i'm now trying to create an upwork acct but always fail as it says that there's no opportunity to get jobs at my expertise field.
how can i overcome this?

**Comment by masaokiba** [comment]
it's because there are too many upwkrs on the platform. they restrict it knowingly. create account in other category and then change it back to web or mobile.
which category is better is hard to tell, try with translator, machine learning, etc...
please answer the questions in david text.

**Comment by KS-Hyon** [comment]
1) yes, i did.
2) i'm working full time. yes, at least at the moment. i check several times a day but can't reply every time. since tomorrow, i can only use git bash.
3) i'm using astril vpn.
4) at kut e-lab but not the same room where i had worked at.
5) i'm a newbie to git thus dunno. even don't know what force push is for.
6) yes, he did.

what is the answer to my requirements? :)

**Comment by masaokiba** [comment]
fyi, i sleep between 1pm to 5 pm, 5am to 9am, PY time.

**Comment by KS-Hyon** [comment]
if so, this is the time to go to bed?

**Comment by KS-Hyon** [comment]
when do u bid?

**Comment by masaokiba** [comment]
but i will stay more for you, just added a tip text. i bid at this hour.

**Comment by masaokiba** [comment]
which country are you creating in?

**Comment by KS-Hyon** [comment]
singapore

**Comment by masaokiba** [comment]
using vpn for upwork isn't very dangerous. it will be killed very soon. they use AI algorithm to detect our style.

**Comment by KS-Hyon** [comment]
i c. i just wanna meet clients there at the moment. i'll apply your approach later if time comes relevant.

**Comment by masaokiba** [comment]
how is it possible to work in KUT btw?

**Comment by KS-Hyon** [comment]
secret :) i'll let alex write a letter about it.

**Comment by masaokiba** [comment]
singapore, japan seem to be blacklisted in upwk with vpn

**Comment by masaokiba** [comment]
moreover, do u have webcam?

**Comment by KS-Hyon** [comment]
what is good then?

**Comment by masaokiba** [comment]
they require video verification when u get a first job.

**Comment by masaokiba** [comment]
that's why we buy account from other local developers.

**Comment by masaokiba** [comment]
it costs $100-$200, but it's difficult to hard this kind of devs.

**Comment by KS-Hyon** [comment]
i won't do project there.
can i also buy such that account?

**Comment by masaokiba** [comment]
do u have webcam? or at least can u prepare?

**Comment by KS-Hyon** [comment]
i've. prepare for what?

**Comment by masaokiba** [comment]
prepare a web cam.

**Comment by masaokiba** [comment]
if you have, try to apply toptal.com, angel.co.

**Comment by KS-Hyon** [comment]
ic.

**Comment by masaokiba** [comment]
it's very long story to describe here right now.

if you want to go with upwk, you need web cam to pass the video verification. what are you gonna do with upwrk?

**Comment by KS-Hyon** [comment]
i'm gonna catch employers at the moment.
it's not high time yet to work devoting all my efforts now.
need to wait by exploring. hehe.

**Comment by KS-Hyon** [comment]
it's a long story too.

**Comment by masaokiba** [comment]
just exploring isn't enough to get a job. unless you feel you're lucky.

**Comment by masaokiba** [comment]
are you available tomorrow? what time?

**Comment by KS-Hyon** [comment]
at the moment, i can't find any other way else than exploring due to environment. 
i pray to be lucky :)

**Comment by KS-Hyon** [comment]
yes, but i can't chat here as i need to install watchdog tomorrow.

**Comment by masaokiba** [comment]
oh dear, i'll prepare the guide asap. and upload the file to this git

**Comment by masaokiba** [comment]
busy nowadays, with improving situations for me.

**Comment by KS-Hyon** [comment]
yes, plz. thx.

**Comment by KS-Hyon** [comment]
what about present?

**Comment by masaokiba** [comment]
i'm sorry, I don't have a chance to get out and do the shopping atm. i'll try to see what i can do for you.

**Comment by KS-Hyon** [comment]
hehe. thx. btw, did u tell my bro as well?

**Comment by KS-Hyon** [comment]
and did you already buy a perfume??

**Comment by masaokiba** [comment]
yes, but the arrived one is the same one as before.

**Comment by masaokiba** [comment]
for hair.

**Comment by KS-Hyon** [comment]
cosmetic set would be better than perfume :)

**Comment by masaokiba** [comment]
i left your bro a message, but no reply yet. he still seems to be under surveillance.

**Comment by KS-Hyon** [comment]
ohhh.
what happened to him?

**Comment by masaokiba** [comment]
not sure, must be because of guards man.

**Comment by KS-Hyon** [comment]
ic

**Comment by KS-Hyon** [comment]
i'm now working around guardsman. hehe

**Comment by KS-Hyon** [comment]
one more thing to tell my bro:
tell him that mum wants several boxes of NIVEA creams that he bought for her last time. 
and tea for dad.

**Comment by KS-Hyon** [comment]
and remind him about mines and my gf's(watch is good if possible)

**Comment by masaokiba** [comment]
wrote him.

**Comment by KS-Hyon** [comment]
good

**Comment by KS-Hyon** [comment]
no file for alex? if yes, plz upload it now. otherwise it's late to get it.

**Comment by masaokiba** [comment]
too busy to prepare anything for alex

**Comment by KS-Hyon** [comment]
ic

**Comment by masaokiba** [comment]
wait for a moment

**Comment by KS-Hyon** [comment]
k

**Comment by masaokiba** [comment]
remember it's my boss's number 151-1411-4893 tell alex

**Comment by KS-Hyon** [comment]
k

**Comment by masaokiba** [comment]
can i remove now?

**Comment by KS-Hyon** [comment]
k

**Comment by KS-Hyon** [comment]
there? can i go to bed?

**Comment by masaokiba** [comment]
yes

**Comment by KS-Hyon** [comment]
cu, bye.

**Comment by KS-Hyon** [comment]
upwk doesn't allow my profile submission. y?

**Comment by masaokiba** [comment]
'cause there are too many similar profiles submitted. did you try with my attachment?

**Comment by KS-Hyon** [comment]
yes, but it didn't work. can you please share me another profile sample with address, user name, country, postal code?

**Comment by masaokiba** [comment]
which country?

**Comment by KS-Hyon** [comment]
russian or australia

**Comment by masaokiba** [comment]
remove only your own comments, not others, otherwise, it will keep the deleted history.

**Comment by masaokiba** [comment]
try with category `translator`. be sure you have no grammatical errors in the overview text.

**Comment by KS-Hyon** [comment]
alex wants any response from u

**Comment by masaokiba** [comment]
I told aunt about his visit. no reply from your bro yet. what does he want esp?

**Comment by masaokiba** [comment]
p8 balance is ready for him.

**Comment by masaokiba** [comment]
we went skiing last weekend, and didn't have time to prepare 4 ur upwk acct creation.

**Comment by KS-Hyon** [comment]
i dont have acct yet as upwk admins rejected all my submissions. :(

**Comment by masaokiba** [comment]
did u read above comments?

**Comment by masaokiba** [comment]
please check david. and when is he leaving?

**Comment by KS-Hyon** [comment]
yes, i checked all. will check david.

**Comment by masaokiba** [comment]
let me know when you downloaded david, I'll reset the git commit history. you will have to remove ur local master branch and then do pull again or do `git reset --hard HEAD~20` on your side.

**Comment by KS-Hyon** [comment]
i checked david.
please do me a favor. i just tried to create an australian upwk acct with machine learning expert but failed again.
I'd like u to write a sample submission so that it can be approved.

**Comment by masaokiba** [comment]
the astrill vpn ip you're using, must be used too many times.
are you ok with Brazilian account?

**Comment by KS-Hyon** [comment]
yes
as long as it can be approved.

**Comment by KS-Hyon** [comment]
write in detail name, address, postal code, professional description, education, employment history, ...

**Comment by masaokiba** [comment]
the problem is you have to do video verification when you get the first job. are you ready for that?

**Comment by KS-Hyon** [comment]
i'll just get employers there not doing jobs.

**Comment by masaokiba** [comment]
what's your skillset to get job with?

**Comment by masaokiba** [comment]
Do you think you can get a cool client with a brand new account? In web and mobile category, it's almost impossible. you need some records of good feedback.

**Comment by masaokiba** [comment]
updated david. contains upk cred which requires video verification, but it can be used for browsing jobs. give it a try. be sure vpn is always connected, close browser tabs when not used. it's a purchased acct but the seller only created the acc and then ran away when video verification is requested by upk.

**Comment by KS-Hyon** [comment]
thx, but what is the title of security answer? "friend's nickname?" or "favorite sports team?"

**Comment by KS-Hyon** [comment]
do u purchase acct? how much do u spend for purchasing one?

**Comment by davidjin0813** [comment]
150-200, idon't remember the question now. it will show up when they ask.

**Comment by davidjin0813** [comment]
were you able to login?

**Comment by KS-Hyon** [comment]
no, i didn't try.

**Comment by KS-Hyon** [comment]
i think if i login here with that acct now, it can be suspended.
i'll keep it now.

**Comment by KS-Hyon** [comment]
and at the moment, it's not allowed to use other's account. just our own accts are allowed.

**Comment by davidjin0813** [comment]
why don't u use astrill vpn now and try?

**Comment by davidjin0813** [comment]
do u have ur team's own upk acct?

**Comment by KS-Hyon** [comment]
watchdog is running. And another guy is working next to me.

**Comment by KS-Hyon** [comment]
we're trying to create upk acct but all failed. Today, 3 resubmissions of AU acct are all failed/

**Comment by KS-Hyon** [comment]
and i'd like u to prepare a new github acct. create and let me know.

**Comment by davidjin0813** [comment]
is this one public?

**Comment by davidjin0813** [comment]
did you try as translator when creating a new account? probably you can't use this vpn for account creation, that ip must be blacklisted.

**Comment by KS-Hyon** [comment]
it could be. security guardman noticed it once.

**Comment by KS-Hyon** [comment]
this vpn? what does it mean? you mean our ip is blacklisted?

**Comment by davidjin0813** [comment]
your ip means astrill ip?

**Comment by KS-Hyon** [comment]
and then shall we use another vpn? what's recommended?

**Comment by KS-Hyon** [comment]
can't i use argentina acct here now using astril then?

**Comment by davidjin0813** [comment]
I'm not sure... recently, we all let the local developers create accounts and then we bought them, so it could be easily done.

**Comment by KS-Hyon** [comment]
are you using astril as well?

**Comment by davidjin0813** [comment]
argentina and brazil ips seems safe now. Japan, australia, hongkong, singapore, some asian countries are blacklisted, because alot of our upworks have tried them

**Comment by KS-Hyon** [comment]
i c. hehe

**Comment by davidjin0813** [comment]
i don't use for my upwork account, i just use it for googling.

**Comment by KS-Hyon** [comment]
i mean, recently after buying local dev's accounts and how did u use it? via astril?

**Comment by davidjin0813** [comment]
not for blacklisted countries, we purchase vps.

**Comment by KS-Hyon** [comment]
ic

**Comment by davidjin0813** [comment]
i'll delete your account completely. do you remember the password of ur email acct here?

**Comment by KS-Hyon** [comment]
i don't remember.

**Comment by KS-Hyon** [comment]
plz try to clear this account not deleting.
and just create another one.

**Comment by davidjin0813** [comment]
hmm, it's even difficult to create a new gmail or outlook without phone number... let me try other services.
are you ok with deleting this account?

**Comment by davidjin0813** [comment]
if you go to https://github.com/, you can see browse activity, and it can't be cleared. that's why i want to delete this account completely.

**Comment by KS-Hyon** [comment]
okay, delete this.

**Comment by KS-Hyon** [comment]
i'll let you know how to avoid phone verify when you create gmail acct.

**Comment by davidjin0813** [comment]
do u know how to?

**Comment by davidjin0813** [comment]
let me know now

**Comment by davidjin0813** [comment]
btw, do u know bitbucket?

**Comment by KS-Hyon** [comment]
install Chrome Extension or Firefox addin which prevents WebRTC.

**Comment by KS-Hyon** [comment]
Chrome Extension : WebRTC Leak Prevent (Icon of this extension seems a flower)

**Comment by davidjin0813** [comment]
is this to use different canvas noise hash?

**Comment by KS-Hyon** [comment]
Firefox Extension : Disable WebRTC (Icon of this extension is like a shield)

**Comment by davidjin0813** [comment]
what is your def stack? web/mobile/ or anything else?

**Comment by KS-Hyon** [comment]
after installing this extension on Chrome, you should select "Disable non-proxied UDP(force only)" in the extension's option.

**Comment by davidjin0813** [comment]
do u know bitbucket?

**Comment by KS-Hyon** [comment]
def stack?
No, recently we also found it difficult to create gmails. Yesterday safeguard man told us that's because of leaking WebRTC. These extensions prevent leaking WebRTC. And now we don't have such difficulty in creating gmails.

**Comment by KS-Hyon** [comment]
did you install those extensions?

**Comment by KS-Hyon** [comment]
try and let me know.

**Comment by davidjin0813** [comment]
yes, i installed.

i meant are you mobile developer? web ? or anything else?

**Comment by KS-Hyon** [comment]
Machine Learning & Computer Vision engineer

**Comment by davidjin0813** [comment]
hmm, just wondering if this kind of jobs are available enough for you to get on upwork... but this kind of engineers are less than other web or mobile developers, so keep your chin up.

**Comment by davidjin0813** [comment]
failed to create a gmail with this plugin turned on

**Comment by KS-Hyon** [comment]
try again

**Comment by KS-Hyon** [comment]
gmail.com doesn't ask phone verify after we install that plugin

**Comment by davidjin0813** [comment]
it doesn't work here...

**Comment by davidjin0813** [comment]
i just created protonmail.

**Comment by KS-Hyon** [comment]
hehe

**Comment by davidjin0813** [comment]
https://github.com/vlad-nicula your new github

email: vlad.nicula318@protonmail.com
username: vlad-nicula
same password
try now and let me know if you confirmed. i'll remove then.

**Comment by davidjin0813** [comment]
ps. email password is the same.

**Comment by KS-Hyon** [comment]
it works. i created a new repository. i'll signout here now.

**Comment by davidjin0813** [comment]
test-project repo is our place.
